Compatibility with Dart 2.10.

trygvis opened the pull request:
This depends on mmcc007/tool_base#2, mmcc007/tool_mobile#1 and mmcc007/fake_process_manager#1.

These changes has been done as a part of getting screenshots to work with Dart 2.10. The new screenshots work in the same way as earlier builds for my app.

trygvis committed:
Compatibility with Dart 2.10.

trygvis commented:
This fixes #215.

mazzonem commented:
@mmcc007 Would you be happy to merge this? #215 seems to be a fairly large issue now that Flutter 2.0 is in Stable.

Thanks in advance 🙏

clangenb commented:
I would also be happy if this is merged.

I think screenshots would be a valuable tool, but the unresponsiveness worries me a bit.

felixwortmann commented:
Is anyone willing to publish a working fork of screenshots if there is no activity soon?

I think this is a great project and it would help a lot of developers.

M123-dev commented:
Ping @mmcc007

teolemon commented:
@trygvis can you try solving the unit tests in the PR so that it's an easier "sell" to merge the PR ? I'm going to try reaching out to @mmcc007

trygvis committed:
Updating apps

trygvis committed:
wip: Updating code.

trygvis committed:
wip: Generating code from screens.yaml

Necessary to replace the 'resources' package.

trygvis committed:
wip: nits

trygvis committed:
wip: Working imagemagick detection

trygvis committed:
wip: small progress

Fixing many small bugs, lots of null pointers and badly converted
code.

trygvis committed:
wip: small progress

Don't hang forever on devices where we're unable to change the locale.

trygvis commented:
@teolemon I've done some work on trying to get it up to date with the latest dart/flutter releases but it is a fair bit of work. Any help and testing would be appreciated. I've worked out a good number of bugs, but I'm sure there are more remaining.

All my work is done on my fork for now.

trygvis committed:
wip: flavors: nullability fixes

trygvis committed:
wip: Adding missing deps

trygvis committed:
wip: Better type annotations

trygvis committed:
wip: modernization

trygvis committed:
wip

Some code is now working, but the response from the Flutter daemon in
startEmulator() doesn't include the emulatorId (emulator-5554) so we
don't know how to talk to adb.

teolemon commented:
Thank you very much @trygvis 🎉 
@mmcc007 does not seem to be reacting. We might opt for a new repository, and publish to pub.dev ?

M123-dev commented:
Sounds good, but this seems to be a very essential plugin, depending on how much time you @trygvis want to invest here you could also ask [fluttercommunity](https://github.com/fluttercommunity) if you could transfer your repository over there. Just as an idea

trygvis committed:
wip

DeanPack commented:
Found this thread here about bringing it up to date. Can I get an update on when this could be useable?

alexxgarci commented:
> Found this thread here about bringing it up to date. Can I get an update on when this could be useable?

Any other package to do the same?
